roles/variables/vars/main.yml

```diff
@@ -504,6 +504,25 @@ kafka_broker_properties:
       confluent.metadata.server.ssl.keystore.type: BCFKS
       confluent.metadata.server.ssl.truststore.type: BCFKS
       confluent.metadata.server.http2.enabled: false
+  rbac_mds_sso:
+    enabled: "{{ rbac_enabled and not external_mds_enabled and sso_mode != 'none' }}"
```

Comment:
@kc596 is SSO is configurable only in case of RBAC is enabled ?

Reply:
Yes, RBAC is pre-requisite for SSO because MDS is not enabled if RBAC is not configured.